README.md

```diff
@@ -18,6 +18,9 @@ SESSION_TIMEOUT_SECONDS=43200
 SESSION_NAME_CLAIMS=userid
 SESSION_TAG_CLAIMS=sub,userid,user_name
 REDIRECT_URIS=https://.....,https://....
+MARKETPLACE_PRODUCT_CODE=xxxxxx
```

Comment:
please update to `MARKETPLACE_PRODUCT_CODE_SC` to match change in the code.

Reply:
done